internal/cmd/pipeline/command_update.go

```diff
@@ -103,6 +116,24 @@ func (c *command) update(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if ksqlCluster != "" {
+		if _, err := c.V2Client.DescribeKsqlCluster(ksqlCluster, environmentId); err != nil {
+			return err
+		}
+		updatePipeline.Spec.SetKsqlCluster(streamdesignerv1.ObjectReference{Id: ksqlCluster})
+	}
+
+	if cmd.Flags().Changed("update-schema-registry") {
+		updateSRCluster, _ := cmd.Flags().GetBool("update-schema-registry")
```

Comment:
Please try to keep a 1-1 mapping between flag names and variables. It makes the code a bit easier to read.
```suggestion
		updateSchemaRegistry, err := cmd.Flags().GetBool("update-schema-registry")
		if err != nil {
			return err
		}
```